python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -439,6 +485,19 @@ async def prepare_for_flat(self, sequence_name: str) -> None:
             )
 
         if calibration_type == CalibrationType.WhiteLight:
+            # Turn off all LEDs
+            await self.rem.ledprojector.cmd_switchAllOff.set_start(
+                timeout=self.long_timeout,
+            )
+            # Confirm that Projector selection is at the LED location
+            vertical_pos = self.linearstage_projector_select.tel_position.get()
```

Comment:
it is a bit dangerous to use `get`.. Since this is telemetry, I would suggest you use `next` with flush=True and a timeout.

```
vertical_pos = await self.linearstage_projector_select.tel_position.next(flush=True, timeout=self.long_timeout)
```